### PR TITLE
Add `4 Digit Player Tournament wiki` and `2024 Tournament Article`

### PR DESCRIPTION
**Pull request by Ekseff**
Tournament wiki for the concluded tournament [4 Digit Player Tournament 6](https://osu.ppy.sh/community/forums/topics/1871322?n=1).

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/4DPT/2024/en.md [hunk 1]
```diff
@@ -0,0 +1,616 @@
...
+| Graphic designer | ::{ flag=SG }:: [[Eun]](https://osu.ppy.sh/users/3788536), ::{ flag=PH }:: [OsuMe65](https://osu.ppy.sh/users/852867), ::{ flag=IN }:: [Raybean](https://osu.ppy.sh/users/16676388), ::{ flag=ID }:: [Dika312](https://osu.ppy.sh/users/741613) |
+| Custom artist | ::{ flag=AU }:: [apexlr8](https://osu.ppy.sh/users/16810724), ::{ flag=AU }:: [Scutoid_621](https://osu.ppy.sh/users/32727820), ::{ flag=FL }:: [naikou_i_guess](https://osu.ppy.sh/users/29139453), ::{ flag=PH }:: [THE BEAT WIZARD](https://osu.ppy.sh/users/17529207), ::{ flag=HK }:: [ShakerSnail](https://osu.ppy.sh/users/14406508) |
+| Wiki editor | ::{ flag=SG }:: [Ekseff](https://osu.ppy.sh/users/13966422), ::{ flag=SG }:: [lovemathboy](https://osu.ppy.sh/users/4220829)  |
+
+## Links
+
+- **[Main website](https://wybin.xyz/tournaments/4dpt6/home)**
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1871322?n=1)
+- [Discord Server](https://discord.gg/PTB8WuVTfB)
```
Comment (iSlodinxOsu) [inline wiki/Tournaments/4DPT/2024/en.md:60]:
```suggestion
- [Discord server](https://discord.gg/PTB8WuVTfB)
```

File: wiki/Tournaments/4DPT/2024/en.md [hunk 2]
```diff
@@ -0,0 +1,616 @@
...
+- [Challonge Bracket](https://challonge.com/4dpt6)
```
Comment (iSlodinxOsu) [inline wiki/Tournaments/4DPT/2024/en.md:61]:
```suggestion
- [Challonge bracket](https://challonge.com/4dpt6)
```

**Review by iSlodinxOsu — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/4DPT/2024/en.md [hunk 3]
```diff
@@ -0,0 +1,616 @@
...
+![4DPT6 banner](img/banner.png)
+
+The **4 Digit Player Tournament Season 6** (***4DPT6***) was a double-elimination 1v1 osu!catch tournament hosted by ::{ flag=SG }:: [Ekseff](https://osu.ppy.sh/users/13966422), ::{ flag=US }:: [Bvyans](https://osu.ppy.sh/users/7789926), ::{ flag=SG }:: [[Eun]](https://osu.ppy.sh/users/3788536) and ::{ flag=ID }:: [Constantine](https://osu.ppy.sh/users/3221898). The tournament was open to players between the rank range of #1,000 - #5,000 in osu!catch. It was the sixth edition of the 4 Digit Player Tournament.
+
+## Tournament Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | 2024-02-01/2024-02-15 |
+| Screening Phase | 2024-02-16/2024-02-22 |
```
Comment (Nivalyx) [inline wiki/Tournaments/4DPT/2024/en.md:19]:
```suggestion
| Screening phase | 2024-02-16/2024-02-22 |
```

File: wiki/Tournaments/4DPT/2024/en.md [hunk 4]
```diff
@@ -0,0 +1,616 @@
...
+| Qualifiers | 2024-02-23/2024-02-25 |
+| Round of 64 | 2024-03-01/2024-03-03 |
+| Round of 32 | 2024-03-08/2024-03-10 |
+| Round of 16 | 2024-03-15/2024-03-17 |
+| Quarter Finals | 2024-03-22/2024-03-24 |
```
Comment (Nivalyx) [inline wiki/Tournaments/4DPT/2024/en.md:24]:
```suggestion
| Quarterfinals | 2024-03-22/2024-03-24 |
```

File: wiki/Tournaments/4DPT/2024/en.md [hunk 5]
```diff
@@ -0,0 +1,616 @@
...
+| Semi Finals | 2024-03-29/2024-03-31 |
```
Comment (Nivalyx) [inline wiki/Tournaments/4DPT/2024/en.md:25]:
```suggestion
| Semifinals | 2024-03-29/2024-03-31 |
```

File: wiki/Tournaments/4DPT/2024/en.md [hunk 6]
```diff
@@ -0,0 +1,616 @@
...
+- [Challonge bracket](https://challonge.com/4dpt6)
+- [Livestream channel A](https://www.twitch.tv/4dpt)
+- [Livestream channel B](https://www.twitch.tv/4dctournies)
+- [Pick'ems page](https://pickem.hwc.hr/tournaments/139) hosted by ::{ flag=DE }:: [hallowatcher](https://osu.ppy.sh/users/1874761)
+
+## Participants
+
+Listed below are the players who have qualified into the Round of 64 (grouped in batches of eight seeds) out of 85 participating players in total.
+
+The full final standings for the Qualifier stage can be found on the following [website](https://wybin.xyz/tournaments/4dpt6/stats#qualifiers).
```
Comment (Nivalyx) [inline wiki/Tournaments/4DPT/2024/en.md:70]:
```suggestion
The full final standings for the Qualifier stage can be found on [this website](https://wybin.xyz/tournaments/4dpt6/stats#qualifiers).
```

File: wiki/Tournaments/4DPT/2024/en.md [hunk 7]
```diff
@@ -0,0 +1,616 @@
...
+  2. [seatrus - Hear+ of Marionette (Mrbinking) \[Puppet+\]](https://osu.ppy.sh/beatmapsets/2160343#fruits/4555591)
+  3. [Kano - Platonic Love (trailmix) \[Astronic's Insane\]](https://osu.ppy.sh/beatmapsets/1122706#fruits/2350361)
+  4. [MiddleIsland - Piano Kyousoukyoku Ni-ban "Nageki no Mai" (kisata) \[Expert\]](https://osu.ppy.sh/beatmapsets/1520799#fruits/3112505)
+- DoubleTime
+  1. [VANTAGE - Count What You Have Now (Crowley) \[MomoX's Rain\]](https://osu.ppy.sh/beatmapsets/1212895#fruits/2638229)
+  2. [Sweet Trip - Chocolate Matter (Bastian) \[Breakup\]](https://osu.ppy.sh/beatmapsets/1503387#fruits/3080020)
+  3. [ChouCho - Kuusou Triangle (CelsiusLK) \[Insane\]](https://osu.ppy.sh/beatmapsets/470971#fruits/1006811)
+  4. [zts - worldenddominator (ktgster) \[End\]](https://osu.ppy.sh/beatmapsets/331034#fruits/733444)
+- Mystery
+  1. [general_r6 - Cupid x Angle Grinder  (e wong) \[cupid if it was a good song\]](https://osu.ppy.sh/beatmapsets/2022815#fruits/4213160) (Sightread)[^sightreading]
```
Comment (Nivalyx) [inline wiki/Tournaments/4DPT/2024/en.md:153]:
```suggestion
  1. *(various maps)* (Sightread)[^sightreading]
```

File: wiki/Tournaments/4DPT/2024/en.md [hunk 8]
```diff
@@ -0,0 +1,616 @@
...
+1. Matches are played in a 1v1 format.
+2. Map scoring is based on **[ScoreV2](/wiki/Gameplay/Score#scorev2).**
+3. Nofail will be enforced during matches and on every map.
+4. For the duration of the tournament, players must be in the discord server at all times.
+5. The 64 players who have qualified from the qualifiers will compete against each other using the double-elimination bracket system.
+
+### Tournament registration
+
+1. In order to participate, players are required to register for the tournament through [the tournament's website](https://wybin.xyz/tournaments/4dpt6/home).
+2. The player must be between the rank range of #1000 to #5000 until the end of registration. After the registration phase has ended, players are allowed to rank up out of the rank range.
```
Comment (Nivalyx) [inline wiki/Tournaments/4DPT/2024/en.md:541]:
```suggestion
2. Each player must be in between the rank range of #1,000 to #5,000 at the end of the registration period. After the registration has ended, players are allowed to rank up or down out of the rank range.
```

File: wiki/Tournaments/4DPT/2024/en.md [hunk 9]
```diff
@@ -0,0 +1,616 @@
...
+3. The player registering should not have any related 4 digit player touranemnt badges from past iterations. This rule is in place to ensure fairness and provide equal opportunities.
```
Comment (Nivalyx) [inline wiki/Tournaments/4DPT/2024/en.md:542]:
```suggestion
3. Each player must not have any related 4-digit player tournament badges from past iterations. This rule is in place to ensure fairness and provide equal opportunities.
```

File: wiki/Tournaments/4DPT/2024/en.md [hunk 10]
```diff
@@ -0,0 +1,616 @@
...
+4. Staff are allowed to play with the exception of Admins, Mappoolers and Custom Mappers. Eliminated players are eligible to be enlisted as Playtesters, Replay Creators and Referees as per the [Official Tourament Support](https://osu.ppy.sh/wiki/en/Tournaments/Official_support) page.
```
Comment (Nivalyx) [inline wiki/Tournaments/4DPT/2024/en.md:543]:
```suggestion
4. Staff members are allowed to play with the exception of Admins, Mappoolers and Custom Mappers. Eliminated players are eligible to be enlisted as Playtesters, Replay Creators and Referees as per the [Official Tourament Support](https://osu.ppy.sh/wiki/en/Tournaments/Official_support) page.
```

File: wiki/Tournaments/4DPT/2024/en.md [hunk 11]
```diff
@@ -0,0 +1,616 @@
...
+
+### Mappool Format
```
Comment (Nivalyx) [inline wiki/Tournaments/4DPT/2024/en.md:545]:
```suggestion
### Mappool format
```

File: wiki/Tournaments/4DPT/2024/en.md [hunk 12]
```diff
@@ -0,0 +1,616 @@
...
+
+1. This iteration features a Mystery Mod, which will be testing on niche skillsets that are not commonly seen in standardised mappools of Catch the Beat. Some examples of these skillsets could include EZ mod, Flashlight, Dodge the beat (DTB), sightreads and so on. It will be up to the creativity of the hosts, mappoolers, custom mappers to curate these skillsets for this particular mod.
+2. Mystery Mod will not be from another gamemode.
```
Comment (Nivalyx) [inline wiki/Tournaments/4DPT/2024/en.md:548]:
```suggestion
2. The Mystery Mod pool will not feature maps that are [converted](/wiki/Beatmap/Converts) from another game mode.
```

File: wiki/Tournaments/4DPT/2024/en.md [hunk 13]
```diff
@@ -0,0 +1,616 @@
...
+3. The mappool format for each of the following rounds are as follows:
```
Comment (Nivalyx) [inline wiki/Tournaments/4DPT/2024/en.md:549]:
```suggestion
3. The mappool format and win condition for each of the following rounds are as follows:
```

File: wiki/Tournaments/4DPT/2024/en.md [hunk 14]
```diff
@@ -0,0 +1,616 @@
...
+- Qualifiers
+  - 3 NM, 2 HD, 2 HR, 2 DT
+- Round of 64, Round of 32 & Round of 16
+  - Best of 9, 1 Ban
+  - 4 NM, 3 HD, 3 HR, 3 DT, 1 Mystery, 1 TB
+- Quarter Finals
```
Comment (Nivalyx) [inline wiki/Tournaments/4DPT/2024/en.md:555]:
```suggestion
- Quarterfinals
```

File: wiki/Tournaments/4DPT/2024/en.md [hunk 15]
```diff
@@ -0,0 +1,616 @@
...
+  - Best of 11, 1 Ban
+  - 4 NM, 3 HD, 3 HR, 3 DT, 2 Mystery, 1 TB
+- Semi Finals
```
Comment (Nivalyx) [inline wiki/Tournaments/4DPT/2024/en.md:558]:
```suggestion
- Semifinals
```

File: wiki/Tournaments/4DPT/2024/en.md [hunk 16]
```diff
@@ -0,0 +1,616 @@
...
+- Finals
+  - Best of 13, 2 Bans
+  - 4 NM, 4 HD, 4 HR, 4 DT, 2 Mystery, 1 TB
+- Grand Finals
+  - Best of 13, 2 Bans
+  - 4 NM, 4 HD, 4 HR, 4 DT, 2 Mystery, 1 TB
+
+### Qualifiers
+
+1. Players will have to sign up for a Qualifier lobby that has been prepared by the tournament hosts listed on the [website](https://wybin.xyz/tournaments/4dpt6/schedule#qualifiers).
```
Comment (Nivalyx) [inline wiki/Tournaments/4DPT/2024/en.md:570]:
```suggestion
1. Each player will have to sign up for a Qualifier lobby that has been prepared by the tournament management as listed on the [tournament's website](https://wybin.xyz/tournaments/4dpt6/schedule#qualifiers).
```

File: wiki/Tournaments/4DPT/2024/en.md [hunk 17]
```diff
@@ -0,0 +1,616 @@
...
+2. Each Qualifier lobby can have a maximum of 16 partipants.
+3. Players are not allowed to ban any beatmaps in the Qualifiers.
+4. In the lobby, players will play through nine Qualifier beatmaps in the mod order of NM, HD, HR and DT.
+5. Players are to join their selected qualifiers lobby on time. In the event you are late to the lobby, you are allowed to reschedule to a later lobby.
```
Comment (Nivalyx) [inline wiki/Tournaments/4DPT/2024/en.md:574]:
```suggestion
5. Players are expected to join their selected qualifiers lobby on time. In the event that a player is late to the lobby, the said player is allowed to reschedule to a later lobby.
```

Comment (Nivalyx) [reply wiki/Tournaments/4DPT/2024/en.md:574]: (the word "you" should be avoided as much as possible while dictating rules)

File: wiki/Tournaments/4DPT/2024/en.md [hunk 18]
```diff
@@ -0,0 +1,616 @@
...
+6. If said player joins their selected qualifiers lobby within the first run, they are allowed to play the remaining maps in the first run and the missed out maps as the second run.
+7. Between each map, all players have **120 seconds** to ready up.
```
Comment (Nivalyx) [inline wiki/Tournaments/4DPT/2024/en.md:576]:
```suggestion
7. Between each map, players have **120 seconds** to ready up.
```

File: wiki/Tournaments/4DPT/2024/en.md [hunk 19]
```diff
@@ -0,0 +1,616 @@
...
+   - After a player reaches 3 points (in the Semi Finals, Finals, and Grand Finals), the opposing player is then given a chance to **ban another beatmap** from the mappool that has neither been picked nor banned up until that point as a comeback mechanic.
+   - A player is not allowed to ban 2 maps from the same mod pool.
+   - No banned beatmaps are allowed to be picked by any player during the remainder of the match.
+   - This rule does not apply in the Qualifiers.
+9. In the event of a tiebreaker, **the tiebreaker beatmap will be played using Hidden or NoMod depending on the player's choice**.
+
+### Scheduling Rules
+1. The hosts will schedule a time for the match. It will take place on a Friday/Weekend unless a special request is to be played Midweek.
+2. In the event that you are unable to make it for your scheduled time, contact your opponent andd apply for a reschedule at the `#reschedule` channel in the tournament's Discord server. Screenshot is required for valid reschedule proof.
+3. Maximum Reschedule time is **6 hours** before your match. For an emergency reschedule or late schedule, it may not be accepted depending on host and referee availability and discretion.
```
Comment (Nivalyx) [inline wiki/Tournaments/4DPT/2024/en.md:610]:
```suggestion
3. Reschedules have to be agreed on at most **6 hours** before the match. Emergency and late reschedules may or may not be accepted depending on the hosts' and the referee's availability and discretion.
```

File: wiki/Tournaments/4DPT/2024/en.md [hunk 20]
```diff
@@ -0,0 +1,616 @@
...
+
+## Notes
+
+[^sightreading] Placeholder map for sightread during the stream and showcase.
+
+[^tester] Enlisted as map tester after being knocked out in Semifinals.
```
Comment (Nivalyx) [inline wiki/Tournaments/4DPT/2024/en.md:616]:
```suggestion
[^tester] This player was brought in as a map tester after being knocked out in the bracket stage.
```

File: wiki/Tournaments/4DPT/2024/en.md [hunk 21]
```diff
@@ -0,0 +1,616 @@
...
+9. In the event of a tiebreaker, **the tiebreaker beatmap will be played using Hidden or NoMod depending on the player's choice**.
+
+### Scheduling Rules
+1. The hosts will schedule a time for the match. It will take place on a Friday/Weekend unless a special request is to be played Midweek.
+2. In the event that you are unable to make it for your scheduled time, contact your opponent andd apply for a reschedule at the `#reschedule` channel in the tournament's Discord server. Screenshot is required for valid reschedule proof.
+3. Maximum Reschedule time is **6 hours** before your match. For an emergency reschedule or late schedule, it may not be accepted depending on host and referee availability and discretion.
+
+## Notes
+
+[^sightreading] Placeholder map for sightread during the stream and showcase.
```
Comment (Nivalyx) [inline wiki/Tournaments/4DPT/2024/en.md:614]:
```suggestion
[^sightreading] Maps for this slot were revealed upon being picked at the match.
```

File: wiki/Tournaments/4DPT/2024/en.md [hunk 22]
```diff
@@ -0,0 +1,616 @@
...
+8. Each player may ban up to two beatmaps from the mappool following a modified "comeback mechanic" system that is adapted from the [osu!catch World Cup 2019](/wiki/Tournaments/CWC/2019). The system works as follows:
+   - At the start of the match, each player has to ban **one beatmap** from the mappool.
+   - After a player reaches 3 points (in the Semi Finals, Finals, and Grand Finals), the opposing player is then given a chance to **ban another beatmap** from the mappool that has neither been picked nor banned up until that point as a comeback mechanic.
+   - A player is not allowed to ban 2 maps from the same mod pool.
+   - No banned beatmaps are allowed to be picked by any player during the remainder of the match.
+   - This rule does not apply in the Qualifiers.
+9. In the event of a tiebreaker, **the tiebreaker beatmap will be played using Hidden or NoMod depending on the player's choice**.
+
+### Scheduling Rules
+1. The hosts will schedule a time for the match. It will take place on a Friday/Weekend unless a special request is to be played Midweek.
```
Comment (Nivalyx) [inline wiki/Tournaments/4DPT/2024/en.md:608]:
```suggestion
1. The hosts will pre-determine a scheduled time for the matches, which will take place on a Friday or a weekend day unless a special request is placed in advance for a match to be played mid-week.
```

File: wiki/Tournaments/4DPT/2024/en.md [hunk 23]
```diff
@@ -0,0 +1,616 @@
...
+2. In the event that you are unable to make it for your scheduled time, contact your opponent andd apply for a reschedule at the `#reschedule` channel in the tournament's Discord server. Screenshot is required for valid reschedule proof.
```
Comment (Nivalyx) [inline wiki/Tournaments/4DPT/2024/en.md:609]:
```suggestion
2. In the event that a player are unable to make it for your scheduled time, said player is advised to contact their opponent and apply for a reschedule at the `#reschedule` channel in the tournament's Discord server. Screenshot is required for valid reschedule proof.
```

File: wiki/Tournaments/4DPT/2024/en.md [hunk 24]
```diff
@@ -0,0 +1,616 @@
...
+| Upper | **[Fuka Pura](https://osu.ppy.sh/users/2326688)** ::{ flag=FR }:: | **1** | 0 | ::{ flag=US }:: [ERA trooperr](https://osu.ppy.sh/users/32028459) | *win by default* |
+
+## Ruleset
+
+### General rules
+
+1. Matches are played in a 1v1 format.
+2. Map scoring is based on **[ScoreV2](/wiki/Gameplay/Score#scorev2).**
+3. Nofail will be enforced during matches and on every map.
+4. For the duration of the tournament, players must be in the discord server at all times.
```
Comment (Nivalyx) [inline wiki/Tournaments/4DPT/2024/en.md:535]:
```suggestion
4. For the duration of the tournament, players are required to be present in the Discord server at all times.
```

File: wiki/Tournaments/4DPT/2024/en.md [hunk 25]
```diff
@@ -0,0 +1,616 @@
...
+| Upper | **[kev aha x](https://osu.ppy.sh/users/19294472)** ::{ flag=GB }:: | **5** | 3 | ::{ flag=PL }:: [Szczyr](https://osu.ppy.sh/users/17329923) | [#1](https://osu.ppy.sh/community/matches/112925727) |
+| Upper | **[Fuka Pura](https://osu.ppy.sh/users/2326688)** ::{ flag=FR }:: | **1** | 0 | ::{ flag=US }:: [ERA trooperr](https://osu.ppy.sh/users/32028459) | *win by default* |
+
+## Ruleset
+
+### General rules
+
+1. Matches are played in a 1v1 format.
+2. Map scoring is based on **[ScoreV2](/wiki/Gameplay/Score#scorev2).**
+3. Nofail will be enforced during matches and on every map.
```
Comment (Nivalyx) [inline wiki/Tournaments/4DPT/2024/en.md:534]:
```suggestion
3. NoFail will be enforced during matches and on every map.
```

**Comment by Nivalyx** [comment]
i think you should add a footnote for every gimmick featured in the Mystery mod pool tbh, as a good amount of them are not really... self-explanatory

File: wiki/Tournaments/4DPT/2024/en.md [hunk 26]
```diff
@@ -0,0 +1,656 @@
...
+7. Between each map, players have **120 seconds** to ready up.
+8. An optional **120 seconds** break will be offered between the first and second run of the mappool.
+9. The second run is optional to all players.
+10. Players will be seeded based on the sum max system, with the top 64 players advancing to the knock-out stages.
+
+### Match Rules
+
+1. If any player does not show up within 15 minutes of the scheduled start time, the match would be deemed as a forfeit for the player that did not show up.
+2. A replay may be initiated if a player faces disturbances within the first 30 seconds of the map. 
+   - For example, malfunctioning peripherals, disconnects and network outages.
```
Comment (Walavouchey) [inline wiki/Tournaments/4DPT/2024/en.md:608]: complete sentence

```suggestion
   - Valid reasons include malfunctioning peripherals, disconnects, network outages, etc.
```

File: wiki/Tournaments/4DPT/2024/en.md [hunk 27]
```diff
@@ -0,0 +1,656 @@
...
+   - This is only applicable once. Lag is not a valid reason.
+3. If any player is caught intentionally stalling a match, displaying poor sportsmanship behaviour, or causing intentional drama during the tournament, the individual will be dealt with accordingly by the hosts.
+
+### Match Procedures
+
+1. Players are required to join the lobby prior to the start of the match. 
+2. Prior to the start of the match, both players will use the `!roll` command to determine the banning and picking order.
+   - The winner of the `!roll` gets to decide who gets the first pick and second ban or vice versa.
+3. There will be no warmups allowed.
+4. Players will be given **120 seconds** for every action (picking & banning). Failure to do so will lead to a random map to be picked by the referee in charge of the match.
```
Comment (Walavouchey) [inline wiki/Tournaments/4DPT/2024/en.md:618]:
```suggestion
4. Players will be given **120 seconds** for every action (picking and banning). Failure to do so will lead to a random map to be picked by the referee in charge of the match.
```

File: wiki/Tournaments/4DPT/2024/en.md [hunk 28]
```diff
@@ -0,0 +1,656 @@
...
+5. Each player is entitled to 1 optional timeout in a match. The duration of this timeout is 120 seconds and it will start after the action timer has run out.
+6. On the Mods HardRock, DoubleTime, the use of Hidden will be allowed as it has no score multipliers.
+7. Players are not allowed to pick or ban the same mode twice in a row.
+8. Each player may ban up to two beatmaps from the mappool following a modified "comeback mechanic" system that is adapted from the [osu!catch World Cup 2019](/wiki/Tournaments/CWC/2019). The system works as follows:
+   - At the start of the match, each player has to ban **one beatmap** from the mappool.
+   - After a player reaches 3 points (in the Semi Finals, Finals, and Grand Finals), the opposing player is then given a chance to **ban another beatmap** from the mappool that has neither been picked nor banned up until that point as a comeback mechanic.
```
Comment (Walavouchey) [inline wiki/Tournaments/4DPT/2024/en.md:624]:
```suggestion
   - After a player reaches 3 points (in the Semifinals, Finals, and Grand Finals), the opposing player is then given a chance to **ban another beatmap** from the mappool that has neither been picked nor banned up until that point as a comeback mechanic.
```

File: wiki/Tournaments/4DPT/2024/en.md [hunk 29]
```diff
@@ -0,0 +1,656 @@
...
+
+### Match Rules
+
+1. If any player does not show up within 15 minutes of the scheduled start time, the match would be deemed as a forfeit for the player that did not show up.
+2. A replay may be initiated if a player faces disturbances within the first 30 seconds of the map. 
+   - For example, malfunctioning peripherals, disconnects and network outages.
+   - This is only applicable once. Lag is not a valid reason.
+3. If any player is caught intentionally stalling a match, displaying poor sportsmanship behaviour, or causing intentional drama during the tournament, the individual will be dealt with accordingly by the hosts.
+
+### Match Procedures
```
Comment (Walavouchey) [inline wiki/Tournaments/4DPT/2024/en.md:612]: headings should be in sentence case

```suggestion
### Match procedures
```

File: wiki/Tournaments/4DPT/2024/en.md [hunk 30]
```diff
@@ -0,0 +1,656 @@
...
+7. Players are not allowed to pick or ban the same mode twice in a row.
+8. Each player may ban up to two beatmaps from the mappool following a modified "comeback mechanic" system that is adapted from the [osu!catch World Cup 2019](/wiki/Tournaments/CWC/2019). The system works as follows:
+   - At the start of the match, each player has to ban **one beatmap** from the mappool.
+   - After a player reaches 3 points (in the Semi Finals, Finals, and Grand Finals), the opposing player is then given a chance to **ban another beatmap** from the mappool that has neither been picked nor banned up until that point as a comeback mechanic.
+   - A player is not allowed to ban 2 maps from the same mod pool.
+   - No banned beatmaps are allowed to be picked by any player during the remainder of the match.
+   - This rule does not apply in the Qualifiers.
+9. In the event of a tiebreaker, **the tiebreaker beatmap will be played using Hidden or NoMod depending on the player's choice**.
+
+### Scheduling Rules
```
Comment (Walavouchey) [inline wiki/Tournaments/4DPT/2024/en.md:630]:
```suggestion
### Scheduling rules
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**